recipes/corrade/all/test_package/CMakeLists.txt

```diff
@@ -2,10 +2,12 @@ cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
+
+find_package(Corrade REQUIRED)
```

Comment:
This pr adds the `Corrace::rc` excutable target.
Do we want to test it?
Does it work with shared libs on Linux/Windows/Macos in cmake?

Reply:
how do you usually test such kinds of targets? do you have a good example?
I mean, you don't usually do `target_link_library(${PROJECT_NAME} <my_executable_target>`

Reply:
I'm not adding an `rc` component, amI? Conan is not creating that target

Reply:
You're creating the target in `corrade-conan-vars.cmake`.

Reply:
https://cmake.org/cmake/help/latest/command/execute_process.html

🤔 

```cmake
get_target_property(CORRADE_RC_PROGRAM Corrade::rc IMPORTED_LOCATION)
execute_process(${CORRADE_RC_PROGRAM} --version) # Idk the command line but just a suggestions
```